servicebroker/lib/servicebroker.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var async = require('asyncawait/async');
+var await = require('asyncawait/await');
+var model = require(path.join(__dirname, './model'))();
+
+var ServiceBroker = function ServiceBroker() {};
+ServiceBroker.prototype.getCatalog = function() {
+  var catalog = JSON.parse(
```

Comment:
Seems that res.sendFile reads the file when each time it is called. Should we read the catalog file only once and cache it?